Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -65,22 +65,29 @@ the token returned by ``kubeadm init``
     name on each node.
 
 For existing installations with ``kube-proxy`` running as a DaemonSet, remove it
-by using the following commands below. **Careful:** Be aware that this will break
-existing service connections. It will also stop service related traffic until the
-Cilium replacement has been installed:
+by using the following commands below.
 
-.. code-block:: shell-session
+.. warning::
+   Be aware that this will break existing service connections. It will also stop service related traffic
```

Comment:
What does `this` refer to?

```suggestion
   Be aware that removing ``kube-proxy`` will break existing service connections. It will also stop service related traffic
```